src/components/TagSet/TagSetFilter.tsx

```diff
@@ -116,7 +116,12 @@ export const TagSetFilter: React.FC<TagSetFilterProps> = chakra(
             buttonType="link"
             data-testid="filter-clear-all"
             id={`ts-filter-clear-all-${id}`}
-            onClick={() => finalOnClick("clearFilters")}
+            onClick={() =>
+              finalOnClick({
+                id: "clear-filters",
+                label: "Clear Filters",
```

Comment:
@charmingduchess let me know what you think of this. I think we also talked about how this static label was not helpful. So now the returned object has an id and label.